app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+let Comment = React.createClass({
+  propTypes: {
+    comment: React.PropTypes.object,
+    level: React.PropTypes.number,
+  },
+
+  getDefaultProps: function() {
+    return {
+      level: 0,
+    };
+  },
+
+  render() {
+    let comment = this.props.comment;
+    let replies = comment.replies;
+    let repliesLength = replies ? replies.length : 0;
+    let commentBoxClasses = 'comment__box ' + (this.props.level % 2 === 0 ? 'comment__box-even' : 'comment__box-odd');
```

Comment:
I recommend using the `classnames` module to create this string. That module makes things a lot better in terms of code readability.

```js
import classNames from 'classnames';

let commentBoxClasses = classNames({
  'comment__box': true,
  'comment__box--even': this.props.level % 2 === 0,
  'comment__box--odd': this.props.level % 2 !== 0,
});
```

The class names should also be `comment__box--even` and `comment__box--odd`, because `even` and `odd` act as modifiers in the BEM naming scheme: https://en.bem.info/method/key-concepts/